test.cpp

```diff
@@ -0,0 +1,73 @@
+/**
+    @brief The program implements a check of the function operation quadro_solve.cpp
+    @version 1.0.0
+*/
+
+#include "quadro_solve.cpp"
+
+/**
+ * @brief A function for reading float values from terminal
```

Comment:
"A function for reading quadratic equation coefficients from terminal" would be more appropriate and descriptive.